packages/ui/src/helpers/authenticator/translations.ts

```diff
@@ -0,0 +1,139 @@
+import {
+  AuthChallengeName,
+  CodeDeliveryDetails,
+  SocialProvider,
+} from '../../types';
+import { translate } from '../../i18n';
+import { AuthenticatorRoute } from './facade';
+
+/**
+ * ConfirmSignIn
+ */
+const getChallengeText = (challengeName: AuthChallengeName): string => {
+  switch (challengeName) {
+    case 'SMS_MFA':
+      return translate('Confirm SMS Code');
```

Comment:
I'm wondering if we can replace all the hard-coded strings throughout this file with the strings in this file:
`packages/ui/src/i18n/dictionaries/authenticator/defaultTexts.ts`

Reply:
Good point, will do.